index.js

```diff
@@ -32,6 +35,8 @@ const checkAuthentication = (config, res, token, ctx, cb) => {
         statusCode = 401;
       }
 
+      console.log(message);
```

Comment:
Just commenting.  Can leave this here, but I'm not sure how useful that will be since we're in the error handling, the function will end up logging the final error message no matter what at this point if auth is required.